src/components/RequestCollaborationButton.jsx

```diff
@@ -42,8 +46,7 @@ export default function RequestCollaborationButton({ otherUserId }) {
   return (
     <Button
       onClick={async () => {
-        const successful = await requestCollaboration(otherUserId);
-        if (successful) refresh();
```

Comment:
`refresh` is no longer called explicitly. instead, we will invalidate the stale query when objects are updated (many examples in code)